File: Scadex.Model/Dtos/Company/Queries/CompanyDto.cs
using Scadex.Core.Model;

namespace Scadex.Model.Dtos.Company.Queries;

public class CompanyDto : IDto
{
    public Guid Id { get; set; }
    public string Name { get; set; } = null!;
    public string? Description { get; set; }

    #region --- IAuditableEntity ---
    public string? CreatedBy { get; set; }
    public string? UpdatedBy { get; set; }
    public DateTime? CreateDateUtc { get; set; }
    public DateTime? UpdateDateUtc { get; set; }
    #endregion

    #region --- IActivatableEntity ---
    public bool IsActive { get; set; }
    #endregion
}

File: Scadex.WebUI/src/models/company/queries/companyDto.ts
/** Ayna: Scadex.Model/Dtos/Company/Queries/CompanyDto.cs */
export interface CompanyDto {
  id: string;
  name: string;
  description: string | null;
  createdBy: string | null;
  updatedBy: string | null;
  createDateUtc: string | null;
  updateDateUtc: string | null;
  isActive: boolean;
}
